src/output-json-alert.c

```diff
@@ -243,6 +243,10 @@ void AlertJsonHeader(const Packet *p, const PacketAlert *pa, json_t *js)
         }
     }
 
+    if (p->flow) {
+        json_object_set_new(js, "app_proto",
```

Comment:
in "print partial flow record or output app_proto" what would control which way to go?